fsharp-backend/tests/httpbytestestfiles/simple-request-headers.test

```diff
@@ -0,0 +1,32 @@
+[http-bytes-handler GET /]
+(let relevantHeaders = Dict.filter_v1 request.headers (fun (k, _v) -> String.startsWith_v0 k "header") in
+let body = relevantHeaders |> toString |> String.toBytes_v0 in
+Http.response body 200)
+
+[request]
+GET / HTTP/1.1
+Host: HOST
+Date: Sun, 08 Nov 2020 15:38:01 GMT
+Content-Length: 7
+header1: value1
+Header2: Value2
+Header-3: Value-3
+Header-with-Casing-conflict: test1
+header-WITH-Casing-conflict: test2
+
+ignored
+
+[response]
+HTTP/1.1 200 OK
+Date: xxx, xx xxx xxxx xx:xx:xx xxx
+x-darklang-execution-id: 0123456789
+Server: darklang
+Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
+Content-Length: 114
+
+{
+  header2: "value2",
+  header1: "value1",
+  header-with-casing-conflict: "test1,test2",
```

Comment:
Would this be your expectation?

Reply:
I've no opinion.

Reply:
This looks good. Worth checking Date, Content-length, Host, etc?